tests/unit/acquisition/test_function.py

```diff
@@ -461,6 +471,175 @@ def prepare_acquisition_function(
     npt.assert_allclose(eci(x), ei(x) * pof(x))
 
 
+class _PseudoTrainableQuadratic(QuadraticMeanAndRBFKernel, PseudoTrainableProbModel):
+    pass
+
+
+class _PseudoTrainableLinear(GaussianProcess, PseudoTrainableProbModel):
+    def __init__(
+        self,
+        *,
+        kernel_amplitude: float | TensorType | None = None,
+    ):
+        kernel = tfp.math.psd_kernels.ExponentiatedQuadratic(kernel_amplitude)
+        super().__init__([lambda x: tf.reduce_sum(x, axis=-1, keepdims=True)], [kernel])
+
+    def __repr__(self) -> str:
+        return "_PseudoTrainableLinear()"
```

Comment:
similarly here: you don't need this class, you can just use
```python
def linear_kernel_gp(kernel_amplitude: float | TensorType | None = None) -> GaussianProcess:
    return GaussianProcess(
        [lambda x: tf.reduce_sum(x, axis=-1, keepdims=True)],
        [tfp.math.psd_kernels.ExponentiatedQuadratic(kernel_amplitude)],
    )
```
since you don't need to inherit from `PseudoTrainableProbModel`